migrations/2025-05-26-092442_mod-change-community-vis/up.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE mod_change_community_visibility
+    DROP COLUMN reason;
```

Comment:
Removed this field because its not practical to set it as part of `update_community`.